tools/webcam/configs/eyes/README.md

```diff
@@ -0,0 +1,31 @@
+# Sunglasses and Bug-eye Effects
+
+We can apply fun effects on videos with pose estimation results, like adding sunglasses on the face, or make the eyes look bigger.
```

Comment:
`like adding sunglasses on the face, or make the eyes look bigger.`
to
`such as adding sunglasses on the face, or making the eyes look bigger.`